### DISCUSSION TITLE
Logo clickable on email

### DISCUSSION THREAD

**Discussion by miltonlm** [dailydotdev/daily#1347] [Feature Requests]
Hello, Thanks for the tool, it's really helpful.

However it would be nice if the logo is clickable and redirects to the page:

![image](https://github.com/user-attachments/assets/30524bae-9017-4df5-8604-bc3ca00ede60)

The logo sent on the daily email.

**Comment by nimrodkra** [comment]
@miltonlm, which page do you want the logo to direct to? Can you please provide the exact URL?

**Reply by miltonlm** [reply]
It should redirect us to app.daily.dev in order to see the daily posts.

**Comment by nimrodkra** [comment]
Awesome, we fixed it. In the next emails you get the logo will be
clickable and send people to app.daily.dev. Thank you for noticing and for
letting us know!
